librz/bin/format/elf/elf_dynamic.h

```diff
@@ -0,0 +1,16 @@
+// SPDX-FileCopyrightText: 2021 08A <08A@riseup.net>
+// SPDX-FileCopyrightText: 2021 RizinOrg <info@rizin.re>
+// SPDX-License-Identifier: LPGL-3.0-only
+
+#include <ht_uu.h>
+#include "elf.h"
+
+#ifndef _INCLUDE_ELF_DYNAMIC_H_
+#define _INCLUDE_ELF_DYNAMIC_H_
+
+typedef struct rz_bin_elf_dt_dynamic {
+	HtUU *info;
+	RzVector *dt_needed;
+} RzBinElfDtDynamic;
```

Comment:
```
../librz/bin/format/elf/elf_dynamic.h:14:3: warning: redefinition of typedef 'RzBinElfDtDynamic' is a C11 feature [-Wtypedef-redefinition]
} RzBinElfDtDynamic;
  ^
../librz/bin/format/elf/elf.h:159:38: note: previous definition is here
typedef struct rz_bin_elf_dt_dynamic RzBinElfDtDynamic; // elf_dynamic.h
                                     ^
```

You should be able to fix this by removing the typedef here and just defining the struct alone:
```c
struct rz_bin_elf_dt_dynamic {
    HtUU *info;
    RzVector *dt_needed;
};
```